.github/workflows/build.yml

```diff
@@ -18,6 +18,14 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         node-version: [lts/*, 17.x, 16.x, 14.x]
```

Comment:
lets change the node versions to lts/*,22,20
this should allow you to delete the include/exclude